src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -508,10 +490,11 @@ private DbContext TryCreateContextUsingAppCode(Type dbContextType, ModelType sta
             {
                 while (ex != null)
```

Comment:
I would create an extension method for Exception to do that, so it could be reused